src/opentime/rationalTime.h

```diff
@@ -135,6 +135,14 @@ class RationalTime {
         return RationalTime {-lhs._value, lhs._rate};
     }
 
+    friend RationalTime operator* (RationalTime lhs, double rhs) {
+        return RationalTime {lhs._value * rhs, lhs._rate};
+    }
+
+    friend RationalTime operator/ (RationalTime lhs, double rhs) {
+        return RationalTime {lhs._value / rhs, lhs._rate};
+    }
+
```

Comment:
We originally omitted overloading `/` and `*` operators for `RationalTIme` because we felt that the meaning of division and multiplication in the context of RationalTime could potentially be ambiguous. Instead we provided `TimeTransform` to make sure it was explicit how these operations work.